### ISSUE TITLE
Error executing command (add-all-removed-blocks)

### ISSUE THREAD

**Issue by ZigTheHedge** [maruohon/worldutils#3]
Hello!

Unfortunately, I cannot get the list of all removed blocks (the same happens with add-all-removed-blocks-from-mod):

```
> worldutils blockreplace blocknamelist add-all-removed-blocks
[12:33:22] [Server thread/INFO]: An unknown error occurred while attempting to perform this command
[12:33:22] [Server thread/WARN]: Couldn't process command: 'worldutils blockreplace blocknamelist add-all-removed-blocks'
java.lang.AbstractMethodError: Method net/minecraftforge/fml/common/registry/FMLControlledNamespacedRegistry.getKeys()Ljava/util/Set; is abstract
        at net.minecraftforge.fml.common.registry.FMLControlledNamespacedRegistry.getKeys(FMLControlledNamespacedRegistry.java) ~[FMLControlledNamespacedRegistry.class:?]
        at fi.dy.masa.worldutils.command.SubCommandBlockReplace.executeBlockNameList(SubCommandBlockReplace.java:296) ~[SubCommandBlockReplace.class:?]
        at fi.dy.masa.worldutils.command.SubCommandBlockReplace.execute(SubCommandBlockReplace.java:191) ~[SubCommandBlockReplace.class:?]
        at fi.dy.masa.worldutils.command.CommandWorldUtils.func_184881_a(CommandWorldUtils.java:90) ~[CommandWorldUtils.class:?]
        at net.minecraft.command.CommandHandler.func_175786_a(CommandHandler.java:104) [k.class:?]
        at net.minecraft.command.CommandHandler.func_71556_a(CommandHandler.java:83) [k.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71333_ah(DedicatedServer.java:423) [ld.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:388) [ld.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]

```
Minecraft: 1.10.2
Forge: 2297

**Comment by maruohon** [comment]
Bah, that one Forge registry bug bites again... it was fixed in 1.11.2 but for some reason not for 1.10.2 >_>

**Comment by maruohon** [comment]
This is now fixed in version 1.10.2-0.3.1 (and this does not affect 1.11.2 or later, since it was fixed in Forge).
